assets/memberships-gate/editor.js

```diff
@@ -2,22 +2,34 @@
 /**
  * WordPress dependencies
  */
-import { __ } from '@wordpress/i18n';
+import { sprintf, __ } from '@wordpress/i18n';
 import { withSelect, withDispatch } from '@wordpress/data';
 import { compose } from '@wordpress/compose';
 import { Fragment, useEffect } from '@wordpress/element';
-import { Button, TextControl, CheckboxControl } from '@wordpress/components';
+import { Button, TextControl, CheckboxControl, SelectControl } from '@wordpress/components';
 import { PluginDocumentSettingPanel } from '@wordpress/edit-post';
 import { registerPlugin } from '@wordpress/plugins';
 
 /**
  * Internal dependencies
  */
+import PositionControl from '../components/src/position-control';
 import './editor.scss';
 
 const styles = [
-	{ name: 'inline', label: __( 'Inline', 'newspack' ) },
-	{ name: 'overlay', label: __( 'Overlay (soon)', 'newspack' ) },
+	{ value: 'inline', label: __( 'Inline', 'newspack' ) },
+	{ value: 'overlay', label: __( 'Overlay', 'newspack' ) },
+];
+
+const overlayPositionsLabels = {
+	center: __( 'center', 'newspack' ),
+	bottom: __( 'bottom', 'newspack' ),
+};
+
+const overlaySizes = [
+	{ value: 'small', label: __( 'Small', 'newspack' ) },
+	{ value: 'medium', label: __( 'Medium', 'newspack' ) },
+	{ value: 'large', label: __( 'Large', 'newspack' ) },
```

Comment:
What about a `full-width` option here, too? On a large screen the "large" size still seems quite small.

<img width="2552" alt="Screen Shot 2023-03-31 at 12 22 54 PM" src="https://user-images.githubusercontent.com/2230142/229199969-604549f5-3194-4d53-a4b9-17e8225e4d91.png">

Maybe within the full-width container we can still restrict the content to the same bounds as the site content itself, so the blocks aren't flush-left with the viewport.

Reply:
01e1ff73f46190dd076b371c072e1f74fb6660aa tweaks the sizes so it reflects the available set in Newspack Campaigns and full width.

Since large is rather large, I set the default to medium now.